.github/workflows/test-db-ops.yaml

```
@@ -25,15 +27,144 @@ on:
+      should_build: ${{ steps.check-branch-existance.outputs.should_build }} # short sha of the commit
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          submodules: 'recursive'
+          token: ${{ secrets.SUBMODULES_TOKEN }}
+          
+      - name: Check if branch exists
+        id: check-branch-existance
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:6:6: Double quote to prevent globbing and word splitting

(shellcheck)